Havit/Havit/Screens/Category/View/CategoryViewController.swift

```diff
@@ -7,22 +7,170 @@
 
 import UIKit
 
-class CategoryViewController: UIViewController {
+import SnapKit
 
+class CategoryViewController: BaseViewController {
+
+    // MARK: - property
+    weak var coordinator: CategoryCoordinator?
+
+    private lazy var categoryCollectionView: UICollectionView = {
+        let layout = UICollectionViewFlowLayout()
+        layout.scrollDirection = .vertical
+
+        let collectionView = UICollectionView(frame: .zero, collectionViewLayout: layout)
+        collectionView.showsVerticalScrollIndicator = false
+        collectionView.register(cell: CategoryCollectionViewCell.self)
+
+        return collectionView
+    }()
+
+    private let categoryCountLabel: UILabel = {
+        let label = UILabel()
+        label.font = .font(.pretendardReular, ofSize: 13)
+        label.text = "전체 0"
+        label.textColor = UIColor.gray003
+
+        return label
+    }()
+
+    // 📌 카테고리 추가 버튼은 재사용될 것 같아서 나중에 따로 빼면 좋을 것 같아요 !
+    private let addButton: UIButton = {
+        var configuration = UIButton.Configuration.plain()
+
+        var container = AttributeContainer()
+        container.font = .font(.pretendardSemibold, ofSize: 12)
+
+        configuration.attributedTitle = AttributedString("카테고리 추가", attributes: container)
+
+        configuration.baseForegroundColor = UIColor.purpleText
+        configuration.image = UIImage(named: "category_add")
+
+        configuration.background.cornerRadius = 23
+        configuration.background.strokeColor = UIColor.purpleLight
+        configuration.background.strokeWidth = 1
+
+        configuration.imagePadding = 2
+        configuration.contentInsets = NSDirectionalEdgeInsets(top: 2, leading: 10, bottom: 3, trailing: 10)
+        configuration.imagePlacement = .leading
+
+        let button = UIButton(configuration: configuration, primaryAction: nil)
+
+        return button
+    }()
+
+    private let backButton: UIButton = {
+        let button = UIButton()
+        button.setImage(UIImage(named: "iconBackBlack"), for: .normal)
+        return button
+    }()
+
+    private let editButton: UIButton = {
+        let button = UIButton()
+        button.setTitle("수정", for: .normal)
+        button.titleLabel?.font = .font(.pretendardMedium, ofSize: 14)
+        button.setTitleColor(UIColor.gray003, for: .normal)
+        return button
+    }()
+
+    // MARK: - life cycle
     override func viewDidLoad() {
         super.viewDidLoad()
+        setDelegation()
+        setNavigationBar()
+    }
+    
+    override func render() {
+        view.addSubViews([categoryCollectionView, categoryCountLabel, addButton])
+
+        categoryCountLabel.snp.makeConstraints {
+            $0.top.equalTo(self.view.safeAreaLayoutGuide.snp.top).inset(22)
```

Comment:
p3;
```suggestion
            $0.top.equalTo(view.safeAreaLayoutGuide.snp.top).inset(22)
```
현재 ViewController위에 올라가있는 view 이외에 다른 view가 없기 때문에 self없이 view.safeAreaLayoutGuide로 써줘도 될 거 같습니다.

Reply:
@YoonAh-dev 넵 ! 반영했습니다 ! ! --> 커밋번호를 따라가세요 슉슉슉슉.. 25fd6f5